GettingStarted.md

```diff
@@ -0,0 +1,103 @@
+# SailfishOS Chum - Getting Started with OBS
+
+This document summarises the proces of building your first package on 
+OBS.  It only skims the surface of OBS and is not a manual.
+
+To start building your packages on OBS you will require an account.
+Follow the process in the README and contact 'lbt' for an account to 
+be set up.
+
+When your account is set up, you will have a "Home Project".  This is where
+you can start creating packages.  The URL for your home project will be:
+https://build.merproject.org/project/show/home:<username>
```

Comment:
```suggestion
https://build.sailfishos.org/project/show/home:<username>
```
Would it be possible to use `https://build.sailfishos.org/project/show/home:` instead? This is now the preferred domain.